app/templates/components/forms/events/view/create-access-code.hbs

```diff
@@ -0,0 +1,83 @@
+    <div class="field">
+      {{ui-radio current='active' name='status' label='Active' value='active'}}
+    </div>
+    <div class="field">
+      {{ui-radio name='status' label='Inactive' value='inactive'}}
+    </div>
+  </div>
+  <div class="inline field">
+    <label class="required">{{t 'Select Tickets applied to the access code'}}</label>
+    <div class="ui hidden divider"></div>
+    {{ui-checkbox label='Select all Ticket types' name='all_ticket_types' checked=allTicketTypesChecked onChange=(action (mut allTicketTypesChecked))}}
+  </div>
+  {{#if allTicketTypesChecked}}
+    <div class="ui list">
+      {{#each data.tickets as |ticketName|}}
+        <div class="item">{{ticketName}}</div>
+      {{/each}}
+    </div>
+  {{/if}}
+  <div>
+    <div class="inline field">
+      {{ui-checkbox class='toggle' label=(t 'Show more options') checked=showMoreOptions onChange=(action (mut showMoreOptions))}}
+    </div>
+    {{#if showMoreOptions}}
+      {{t 'Set the min and max quantity allowed to purchase per order(Optional)'}}
+      <div class="{{unless device.isMobile 'fields'}}">
+        <div class="wide field {{if device.isMobile 'sixteen' 'five'}}">
+          <label>{{t 'Min'}}</label>
+          {{input type='number' name='min' min=0 value=data.minQuantity}}
+        </div>
+        {{#if device.isMobile}}
+          <div class="ui hidden divider"></div>
+        {{/if}}
+        <div class="wide field {{if device.isMobile 'sixteen' 'five'}}">
+          <label>{{t 'Max'}}</label>
+          {{input type='number' name='max' min=0 value=data.maxQuantity}}
+        </div>
+      </div>
+      <div class="ui hidden divider"></div>
+      {{t 'Validity Period(Optional)'}}
+      <div class="fields">
+        <div class="wide field {{if device.isMobile 'sixteen' 'five'}}">
+          <label>{{t 'Valid from'}}</label>
+          {{widgets/forms/date-picker id='start_date' value=data.validFromDate}}
+          <div class="ui hidden divider"></div>
+          {{widgets/forms/time-picker id='start_time' value=data.validFromTime}}
+        </div>
+        <div class="wide field {{if device.isMobile 'sixteen' 'five'}}">
+          <label>{{t 'Expires on'}}</label>
+          {{widgets/forms/date-picker id='end_date' value=data.validTillDate}}
+          <div class="ui hidden divider"></div>
+          {{widgets/forms/time-picker id='end_time' value=data.validTillTime}}
+        </div>
+      </div>
+    {{/if}}
+    <div class="field">
+      <label>{{t 'Access Link'}}</label>
+      <div class="ui action input fluid">
+        <input type="text" value="{{accessUrl}}" readonly>
+        {{#copy-button
+          clipboardText=accessUrl
+          class='ui black right labeled icon copy button'}}
+          <i class="copy icon"></i>
+          {{t 'Copy'}}
+        {{/copy-button}}
+      </div>
+    </div>
+  </div>
+  <div class="ui hidden divider"></div>
+  <button class="ui button">{{t 'Cancel'}}</button>
```

Comment:
Actually ... No cancel button needed here. This isn't a modal anyway. Just a page